LFGBulletinBoard/GroupBulletinBoard.lua

```diff
@@ -373,6 +373,16 @@ end
 -- Tag Lists
 --------------------------------------------------------------------------------
 local tagCollisions
+local shouldUpdateTagKey = function(pattern, current, incoming)
+	assert(incoming, "shouldUpdateTagKey: incoming key is nil", pattern, current, incoming)
+	if current == incoming then return false end
+	if not current then return true end
+	incoming = GBB.GetDungeonInfo(incoming)
+	current = GBB.GetDungeonInfo(current); 
+	current = current and current.expansionID or -1
+	incoming = incoming and incoming.expansionID or -1
```

Comment:
any key without matching info/expansionID will have lowest priority for the pattern atm. (ie custom filters)